content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -3,135 +3,185 @@ title: Instrumentation
 weight: 3
 ---
 
-This guide will cover creating and annotating spans, creating and annotating metrics, how to pass context, and a guide to automatic instrumentation for JavaScript. This simple example works in the browser as well as with Node.js
+Manual instrumentation is the process of adding observability code to your
+application.
 
-## Example Application
+## Initialize Tracing
 
-In the following this guide will use the following sample app:
+To start [tracing](/docs/concepts/signals/traces/#tracing-in-opentelemetry),
+you'll need to have an initialized
+[`TracerProvider`](/docs/concepts/signals/traces/#tracer-provider) that will let
+you create a [`Tracer`](/docs/concepts/signals/traces/#tracer).
 
-```javascript
-'use strict';
+### Node.js
 
-for (let i = 0; i < 10; i += 1) {
-  doWork();
-}
+To initialize tracing with the Node.js SDK, first ensure you have the SDK
+package and OpenTelemetry API installed:
 
-function doWork() {
-  console.log('work...');
-  // simulate some random work.
-  for (let i = 0; i <= Math.floor(Math.random() * 40000000); i += 1) {}
-}
+```shell
+npm install \
+  @opentelemetry/api \
+  @opentelemetry/sdk-node
 ```
 
-## Initializing a Tracer
+It's recommended to have a separate `tracing.js` file that has all the SDK
+initialization code in it:
+
+```js
+import opentelemetry from "@opentelemetry/api";
+import nodeOTel from "@opentelemetry/sdk-node";
+
+const sdk = new nodeOTel.NodeSDK({
+  traceExporter: new opentelemetry.tracing.ConsoleSpanExporter(),
```

Comment:
For performance-sensitive scenarios, I would assume it's better to use sdk-trace-base?

Reply:
Indeed but i think most users still want to use the node-sdk because of the auto instrumentation, i was just saying that we might want to document that it has a performance impact

Reply:
Are there figures somewhere around what the rough overhead might look like? I'm hesitant to say there's a performance impact without an idea of what kind of impact.

Reply:
The overhead depend on the application. If it is using promises/await a lot and has mostly no CPU intensive tasks overhead is higher. Therefore it's impossible to provide any numbers.

> For performance-sensitive scenarios, I would assume it's better to use sdk-trace-base?

Not that instrumentations depend on a working context manager as they get current active span from context. Therefore if one decides to improve performance by using sdk-trace-base they have to be aware that most instrumentations wont work or the user has to take care to implement a context manager.